wirtschaftsuniversitat-wien-apa-based.csl

```diff
@@ -2,20 +2,20 @@
 <style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" name-delimiter="/" initialize="false" initialize-with="." demote-non-dropping-particle="never">
   <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
   <info>
-    <title>Wirtschaftsuniversität Wien - Master Management</title>
-    <id>http://www.zotero.org/styles/wirtschaftsuniversitat-wien-master-management</id>
-    <link href="http://www.zotero.org/styles/wirtschaftsuniversitat-wien-master-management" rel="self"/>
+    <title>Wirtschaftsuniversität Wien - APA Based (English and German)</title>
```

Comment:
@georgd, would `<title>Wirtschaftsuniversität Wien - APA</title>` be acceptable? We never include languages in the title for multi-lingual styles, and "Based" is a bit redundant.